Classes/Plugin.php

```diff
@@ -111,7 +111,7 @@ public function request($uri)
             return strpos($page->getUrl(), self::$uri) !== false and strpos($page->getFilePath(), 'index') === false;
         };
 		$repo = new \Phile\Repository\Page($this->settings);
-        $pages = array_filter($repo->findAll(), $filter);
```

Comment:
findAll() no longer returns an array, but a PageCollection. Luckily it still has a toArray() function.